### PR TITLE
Adjust styling for figures

### PR DESCRIPTION
**Pull request by thatbudakguy**
These are mostly small changes intended to enhance readability; feel free to accept or reject piecemeal as you see fit.

I didn't do anything to figure 4 pending new designs from Gissoo on that one.

- Replace picture of le/yue character with actual text
- Shrink figure 1's height slightly
- Adjust positioning for figure 2
- Adjust text styles for figure 3 and figure 7
- Add caption and source for figure 5
- Adjust spacing and caption for figure 6

### CODE REVIEW

File: content/issues/4/unnatural-language/index.md
Comment (rlskoeser) [inline content/issues/4/unnatural-language/index.md:32]: I noticed when I implemented the first draft of your figures that our deepzoom shortcode doesn't support captions, but it seems like it should and would not be difficult to add. That way the captions will be semantically associated with the images and have consistent formatting. Do you agree?

**Review by rlskoeser — APPROVED** [review]
Thank you for these revisions, @thatbudakguy ! Everything here looks good to me.

I didn't do too much with the figures because we're still waiting on Gissoo's design work, so I just did a quick first pass to get a draft of the article with figures on the develop site.  I did a quick, minimal implementation of the styles for figure 6 but plan to revisit and implement as designed.

File: content/issues/4/unnatural-language/index.md
Comment (thatbudakguy) [reply content/issues/4/unnatural-language/index.md:359]: no, i changed it! sorry, i should've mentioned that. i just picked a two-page spread that i thought better illustrated the big character/small character printing style used for headwords/annotations.

File: content/issues/4/unnatural-language/index.md
Comment (thatbudakguy) [reply content/issues/4/unnatural-language/index.md:32]: yes, i was thinking the same thing!

File: content/issues/4/unnatural-language/index.md
Comment (rlskoeser) [reply content/issues/4/unnatural-language/index.md:359]: no problem! I thought it looked different when I glanced at it - thanks for explaining and for updating, the new image looks great.

File: content/issues/4/unnatural-language/index.md
Comment (rlskoeser) [reply content/issues/4/unnatural-language/index.md:32]: I'll make an issue and update your deep zoom figure captions once we support it

File: content/issues/4/unnatural-language/index.md
Comment (thatbudakguy) [inline content/issues/4/unnatural-language/index.md:108]: oops, this isn't a book 🤦🏻  but i guess the same styles should apply. i did this to prevent the default styling for blockquote sources, which prepends a `- `, because i (mostly) use `<cite>` to semantically mark book titles in the figures/interludes

Comment (rlskoeser) [reply content/issues/4/unnatural-language/index.md:108]: ah, I was wondering. Could we scope the style to interludes or specific figures?

Comment (thatbudakguy) [reply content/issues/4/unnatural-language/index.md:108]: yeah, that should work fine.

**Comment by gwijthoff** [comment]
The color of hyperlinks in interludes disappears against the purple:

Screenshot 2023-09-07 at 9 20 52 AM

So I changed the color to match the main body text, retaining the underline to indicate a link.

File: content/issues/4/unnatural-language/index.md
Comment (gwijthoff) [inline content/issues/4/unnatural-language/index.md:144]: @thatbudakguy for some reason, this cite tag introduces a line break on the page between "The" and "Documents". Also it seems strange to have the second line break when the text is phrased as a sentence: "The / documents / state..."

Screenshot 2023-09-07 at 9 34 45 AM

**Review by gwijthoff — CHANGES_REQUESTED** [review]
Thanks so much for these tweaks, @thatbudakguy! Love the new two-page spread from the _Jingdian Shiwen._ Two brief notes, one inline above, the other below. Not sure if this one is for you or @rlskoeser:

Is there a way to center the table in Figure 2 so that it's in the middle of the page, without center-aligning the text in each table cell? Essentially just bumping the table over to the right so that there's equal padding on either side of it.

**Comment by rlskoeser** [comment]
> Is there a way to center the table in Figure 2 so that it's in the middle of the page, without center-aligning the text in each table cell? Essentially just bumping the table over to the right so that there's equal padding on either side of it.

@gwijthoff this is me! I haven't implemented the full styles for figure 2 as designed, I just dropped in a draft version with very preliminary styles. I don't think I have an issue yet to track this work; I'll go ahead and create one.

**Comment by rlskoeser** [comment]
@gwijthoff what changes do you want to see before you're ready to merge this? I will do figure 2 styles in a separate PR and don't want to hold this up

**Comment by gwijthoff** [comment]
@rlskoeser That makes sense. I can create a separate issue for the [line break thing above](https://github.com/Princeton-CDH/startwords/pull/326#discussion_r1318622261), or we can resolve it here. Up to you!

File: content/issues/4/unnatural-language/index.md
Comment (gwijthoff) [reply content/issues/4/unnatural-language/index.md:144]: Strange, I was serving your branch locally and that's what I got. But I just did a hard refresh and the line breaks are gone. Sorry for the confusion

**Review by gwijthoff — APPROVED** [review]

**Pull request merged**
